tdp_vars_defaults/hue/hue.yml

```
@@ -0,0 +1,128 @@
+# Copyright 2022 TOSIT.IO
+# SPDX-License-Identifier: Apache-2.0
+
+---
+# Hue user and group
+hue_user: hue
+hdfs_user: hdfs
+hadoop_group: hadoop
+
+# Hue version
+hue_release: hue-release-4.10.0
```

```suggestion
hue_release: hue-4.10.0
```

The release name from the link you provided is different (and created folder from unarchiving also differs)

---

Readme is updated with the right link. Readme content will be updated with more information.